src/Utils/arm/request.ts

```diff
@@ -66,7 +66,7 @@ export async function armRequest<T>({
   const response = await window.fetch(url.href, {
     method,
     headers: {
-      Authorization: userContext.authorizationToken
+      Authorization: !userContext.authorizationToken ? "" : userContext.authorizationToken
```

Comment:
This is not correct. We must always have an authorization token set. Can we throw an error earlier in the function?

Something like: `if (!userContext.authorizationToken) { throw new Error("No token provided") }`

Reply:
OK,I will fix that